Onboarding/How-To-Use-GitHub-To-Create-Edit-EOS-Documentation.md

```diff
@@ -0,0 +1,145 @@
+# How To Create and Edit EOS Documentation in GitHub
```

Comment:
Just to clarify that this procedure is for internal DevRel members who have push access to the ENF repos. However, it can still be valuable for another document targeted for external contributors who don't have push access to our ENF repos but who would still like to contribute directly. Most likely, they will have to fork the target repo first and create the PR using their forked source branch. This is the procedure they need to follow (after adapting/including some of the steps below for our case): https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

Reply:
Do we need another document for a different audience for the community?

Reply:
Not really since we have docs for external contributors already.